daffodil-test/src/test/scala/org/apache/daffodil/extensions/TestdoubleToRawLong.scala

```diff
@@ -0,0 +1,42 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.daffodil.extensions
+
+import org.junit.Test
+import org.junit.Test
+import org.apache.daffodil.tdml.Runner
+import org.junit.AfterClass
+
+object TestdoubleToRawLong {
+  val testDir = "/org/apache/daffodil/extensions/doubleToRawLong/"
+
+  val runner = Runner(testDir, "doubleToRawLong.tdml", validateTDMLFile = false)
+
+
+  @AfterClass def shutDown: Unit = {
+    runner.reset
+
+  }
+
+}
+
+class TestdoubleToRawLong {
+  import TestdoubleToRawLong._
+  @Test def doubleToRawLong(): Unit = { runner.runOneTest("doubleToRawLong") }
+
+}
```

Comment:
Since each .tdml/.scala file only has a single test, I would suggest combining them into a single file. There's not much benefit in separating them, and just adds extra test suites/files to manage. In fact, it might even be better to just include them in an existing .tdml/.scala file. `Functions.tdml` and `TestDFDLExpressions.scala` might be a reasonable place to put these tests.